src/components/data_portal/DataPortalShared.tsx

```diff
@@ -26,13 +26,13 @@ export function useLocalStorage(key, initialValue) {
 
 //nicer ajaxRequest
 export function ajaxRequest(parameters) {
-  var method = parameters.method ? parameters.method : "GET"
-  var data = parameters.data
+  let method = parameters.method ? parameters.method : "GET"
+  let data = parameters.data
     ? typeof parameters.data === "object"
       ? JSON.stringify(parameters.data)
       : parameters.data
     : ""
-  var xmlhttp = new XMLHttpRequest()
+  let xmlhttp = new XMLHttpRequest()
```

Comment:
See if you can use `eslint` or `tslint` with `prettier`. It should be included in this project's `package.json` already.